packages/gatsby-plugin-postcss/src/gatsby-node.js

```diff
@@ -0,0 +1,60 @@
+exports.onCreateWebpackConfig = (
+  { actions, stage, loaders, getConfig },
+  { postcss: postcssOptions = {} } = {}
+) => {
+  const isProduction = stage !== `develop`
+  const isSSR = stage.includes(`html`)
+  const originalConfig = getConfig()
+
+  originalConfig.module.rules = originalConfig.module.rules.filter(rule => {
```

Comment:
this would be unexpected behavior to me, plguins should be additive here I think, I would really dislike if if this plugin removed my loaders elsewhere, especially since this doesn't always replace it with something else. 

e.g. if I have `gatsby-plugin-sass` installed with a postcss config meant to augment my sass workflow, this plugin would remove that, and then replace it with a loader that doesn't handle `.scss` files.

 at the very least this should _only_ target the gatsby css-loader

Reply:
@jquense Great catch, I will fix that.